### DISCUSSION TITLE
no access to copilot chat

### DISCUSSION THREAD

**Discussion by VitalieZ** [community/community#106896] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I activated the copilot and it works fine, but in the copilot chat I have a notification "Your Copilot subscription has expired.". For copilot I use one account and for git pull request I use another account, probably from here the problem. Does anyone know how to solve this problem? Thank you!

**Comment by davecheney** [comment, marked as answer]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in. When it does, use your @VitalieZ identity.
